src/lib/components/workflow/start-workflow-search-attributes.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import Button from '$lib/holocene/button.svelte';
+  import {
+    customSearchAttributes,
+    type SearchAttributeInput,
+  } from '$lib/stores/search-attributes';
+
+  import StartWorkflowSearchAttributeInput from './start-workflow-search-attribute-input.svelte';
+
+  export let attributesToAdd: SearchAttributeInput[] = [];
+
+  const addSearchAttribute = () => {
+    attributesToAdd = [...attributesToAdd, { attribute: '', value: '' }];
+  };
+
+  $: searchAttributes = [...Object.entries($customSearchAttributes)].map(
+    ([key, value]) => ({ label: key, value: key, type: value }),
+  );
+
+  const onRemove = (attribute: string) => {
+    attributesToAdd = attributesToAdd.filter((a) => a.attribute !== attribute);
+  };
+</script>
+
+{#each attributesToAdd as attribute}
+  <StartWorkflowSearchAttributeInput
+    {attributesToAdd}
+    {searchAttributes}
+    bind:attribute
+    {onRemove}
+  />
+{/each}
+<Button
+  variant="ghost"
+  leadingIcon="add"
+  on:click={addSearchAttribute}
+  disabled={!searchAttributes.length ||
```

Comment:
Wondering if we might want some kind of tooltip when there are no search attributes and this is disabled?

![Screenshot 2024-06-24 at 11 12 25 AM](https://github.com/temporalio/ui/assets/15069288/12b814fa-7e1f-4f1a-b2e6-5b9141823e84)

Reply:
Great idea, but not gonna do it at the moment. I think we'll get some more feedback on SAs in general and will make a bigger change.